custom_components/sector/binary_sensor.py

```diff
@@ -27,6 +28,41 @@ async def async_setup_entry(
     devices = coordinator.data.get("devices", {})
     entities = []
 
+    # Define descriptions for each type of binary sensor using the generic description
+    SENSOR_DESCRIPTIONS = (
+        GenericSensorEntityDescription(
```

Comment:
```suggestion
        BinarySensorEntityDescription(
```
Use from `from homeassistant.components.binary_sensor import BinarySensorEntityDescription`
We don't need to expand with own keys.

Reply:
Updated.